cms/check_free_space

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+# Copyright 2012-2020 CERN
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Vincent Garonne, <vincent.garonne@cern.ch>, 2015
+# - Donata Mielaikaite, <donata.mielaikaite@cern.ch>, 2020
+# - Fernando Garzon, <oscar.fernando.garzon.miguez@cern.ch>, 2021
+
+'''
+Probe to check the free space at each rse.
+'''
+
+import sys
+import traceback
+from rucio.core import monitor
+from rucio.core.rse import list_rses, get_rse_usage
```

Comment:
If you can use rucio.api.rse functions that would be even better. There is often a 1:1 map.